src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -51,7 +51,7 @@ def _process_file(
         validate_error_list.append(lca_error)
 
     transpiler_result: TranspilationResult = _transpile(
-        transpiler, config.source_dialect, config.target_dialect, source_sql, input_file, []
+        transpiler, config.source_dialect or "", config.target_dialect, source_sql, input_file, []
```

Comment:
Now that this is required, there's no need to check for empty

Reply:
done